packages/migrate/migrations/routes/migrate_server/index.js

```diff
@@ -30,102 +30,104 @@ export function migrate_server(content) {
 
 	for (const statement of file.ast.statements) {
 		for (const method of methods) {
-			const fn = get_function_node(statement, file.exports.map.get(method));
-			if (fn) {
+			const fn = get_function_node(statement, /** @type{string} */ (file.exports.map.get(method)));
+			if (fn?.body) {
 				rewrite_returns(fn.body, (expr, node) => {
-					// leave `() => new Response(...)` alone
-					if (is_new(expr, 'Response')) return;
-
-					const nodes = ts.isObjectLiteralExpression(expr) && get_object_nodes(expr);
-
-					if (nodes) {
-						const body_is_object_literal = nodes.body && ts.isObjectLiteralExpression(nodes.body);
-
-						let safe_headers = !nodes.headers;
-						if (nodes.headers) {
-							if (ts.isObjectLiteralExpression(nodes.headers)) {
-								// if `headers` is an object literal, and it either doesn't contain
-								// `set-cookie` or `set-cookie` is a string, then the headers
-								// are safe to use in a `Response`
-								const set_cookie_value = nodes.headers.properties.find((prop) => {
-									return (
-										ts.isPropertyAssignment(prop) &&
-										ts.isStringLiteral(prop.name) &&
-										/set-cookie/i.test(prop.name.text)
-									);
-								});
-
-								if (!set_cookie_value || is_string_like(set_cookie_value)) {
-									safe_headers = true;
+					if (expr) {
```

Comment:
what if we changed the implementation of `rewrite_returns` instead? I don't think there's any reason we'd need to do anything with a `return;`, we can just leave it be:

```diff
-if (ts.isReturnStatement(node)) {
+if (ts.isReturnStatement(node) && node.expression) {
```
```

Reply:
Depends on how we answer the questions in the PR description

Reply:
At present the type of `Load` is this: https://github.com/sveltejs/kit/blob/ac5b00bd477b35f1c90cb31418b41a8ef0502589/packages/kit/types/index.d.ts#L180-L186

i.e. a `void` return is fine, and so assuming we're okay with that then `return;` is fine

Reply:
Right, but currently the `RequestHandler` type does not allow a `void` return - which makes me think that we should probably align these. So `Load` is ok with a `void`, so `GET` should, too. In the case of standalone endpoints we always need to return a response now. If we assume that people have their returns correctly already, we can just skip the `return;` case - if you agree with that, I'll adjust the code accordingly.

Reply:
If standalone endpoint returns void today, it'll be an error, so I think it's fine for `RequestHandler` to require a `Response` return value — it'll just continue to error if it doesn't

Reply:
True - ok I'll adjust this to leave lone returns as-is then